packages/devnet/genesis.json

```diff
@@ -0,0 +1,60 @@
+{
+  "config": {
+    "chainId": 22395,
```

Comment:
Why that number? If there is no reason - let's use 50. That's the network id we're using for testing everywhere else.

Reply:
No reason. This was randomly generated by puppeth. Just changed it to 50.